src/index.css

```diff
@@ -79,9 +79,21 @@ html {
   caret-color: var(--primary);
 }
 html[bg-theme='404'] {
-  background-position: top, center, top;
-  background-size: auto auto, 100% 100%, 3rem 3rem;
-  background-image: url(), url(), var(--bg-grid);
+  background-position: center, center, center, top;
```

Comment:
Are you sure that it's better to use plain css instead of `Background` component?